build_tools/benchmarks/comparisons/setup_desktop.sh

```diff
@@ -1,34 +1,40 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
 #!/bin/bash
 
+set -eu pipefail
```

Comment:
Probably because it means the pipefail option was actually set and you had some command in a pipe that returned a non-zero exit code. The most common culprit in that case for "not really an error" is grep, in my experience.